src/EnergyPlus/Data/EnergyPlusData.hh

```diff
@@ -68,6 +79,107 @@ namespace EnergyPlus {
 //    }
 //};
 
+    struct BoilersData : BaseGlobalStruct {

```

Comment:
Most of these boilers and chiller data classes only contain three members: a ```getInput``` flag, an integer ```numObjects``` member, and a 1D array of objects. These should probably get consolidated later.

Reply:
This is going to prove to be exceedingly common as we move through plant components.